src/ReachabilityAnalysis.jl

```diff
@@ -24,7 +24,8 @@ include("Continuous/fields.jl")
 include("Continuous/normalization.jl")
 include("Continuous/homogeneization.jl")
 include("Continuous/linearization.jl")
-include("Discretization/exponentiation.jl")
+include("Discretization/ExponentiationModule.jl")
```

Comment:
I would put modules in their own folder.
I would also not write "Module" in their name. Otherwise, should each module have that in its name?

Reply:
yes, modules defining data structures (or groups of related) utilities would have "..Module" them. the exception would be modules primarily *relating* them. we can have a chat next dev call, where i can be elaborate about the specifics.

Reply:
maybe we merge this, then in the next dev call write down the nomenclatures to be applied in the refactor?